test/integration/narrow/routes/capture.test.js

```diff
@@ -0,0 +1,81 @@
+describe('Capture test', () => {
+  jest.mock('ffc-messaging')
+  jest.mock('../../../../app/plugins/crumb')
+  const createServer = require('../../../../app/server')
+  let server
+  const url = '/capture'
+
+  jest.mock('../../../../app/debt')
+  const { getDebts } = require('../../../../app/debt')
+
+  const debts = [{
+    scheme: 'SFI Pilot',
+    frn: '1234567890',
+    reference: 'SFIP1234567',
+    netValue: 1000.00,
+    debtType: 'Irregular',
+    recoveryDate: '19/01/2022',
+    attachedDate: '',
+    createdBy: 'John Watson'
+  },
+  {
+    scheme: 'SFI',
+    frn: '1234567891',
+    reference: 'SFIP1234568',
+    netValue: 570.00,
+    debtType: 'Administrative',
+    recoveryDate: '18/01/2022',
+    attachedDate: '18/01/2022',
+    createdBy: 'Steve Dickinson'
+  }]
+
+  beforeEach(async () => {
+    getDebts.mockResolvedValue(debts)
+    server = await createServer()
+    await server.initialize()
+  })
+
+  afterEach(async () => {
+    jest.resetAllMocks()
+    await server.stop()
+  })
+
+  describe('GET requests', () => {
```

Comment:
This is great and something we should encourage and do

Reply:
I think it would be good to get our collective heads together to come up with best practice. There is some cool stuff that peeps have been doing with test, but we have a lot of inconsistencies too.